src/prefect_server/graphql/schema/runs.graphql

```diff
@@ -140,3 +147,7 @@ type mapped_children_payload {
   max_end_time: DateTime
   state_counts: JSON
 }
+
+type deleted_count_payload {
+  affected_rows: Int
+}
```

Comment:
A little uncertain about best practice here

Reply:
I'm a little more in favor of using the standard `success` payload that the other functions. Especially if the delete on a list is an atomic operation then it either deleted all of them or it didn't.

Reply:
It's not atomic -- does that change your mind?

Reply:
Yeah, affected rows is fine then because it's the most convenient way to convey information on how "successful" the delete was

Reply:
This could be moved to a generic `affected_rows` payload